### ISSUE TITLE
Color field accessibility

### ISSUE THREAD

**Issue by missmatsuko** [craftcms/cms#7320]
### Description
<img width="229" alt="Screenshot of color fields." src="https://user-images.githubusercontent.com/17884198/102928884-66eedc80-444e-11eb-8147-1de6f62e9b25.png">

- It's not clear that the text input is for hex color codes. Maybe visually prefix the input with a hash, like the number field with a prefix? 
- The color picker input is missing a label
- May want to link the color picker input and text input with `aria-controls`?

### Steps to reproduce
1. Create and inspect a color field.

### Additional info
- Craft version: 3.5.17.1

**Comment by missmatsuko** [comment]
Hmm seems like the browser native color input is not keyboard-accessible in Safari:
https://css-tricks.com/color-inputs-a-deep-dive-into-cross-browser-differences/

**Comment by brandonkelly** [comment]
Resolved for the next release! 2171ad788871e09f1e7909b1f85656a0c5eb1787
